chxtools/chx_wrapper.py

```diff
@@ -16,19 +17,19 @@ def det_select(det):
     calling sequence: det_select(det)
     """
     try:
-	rm_det=ascan.user_detectors[0].name
-	ascan.user_detectors.remove(session_mgr[rm_det])
+        rm_det=ascan.user_detectors[0].name
```

Comment:
This is using _very old_ ophyd API that definitely no longer works. It took me a minute to even remember `.user_detectors`; I think it goes back to the days when the RunEngine was part of ophyd and there was no such thing as plans.  @mrakitin Can you open an issue to revisit this function?